rfps/polkadot-collator-setup.md

```diff
@@ -0,0 +1,39 @@
+# Polkadot Collator Setup
+
+* **Status:** Open
+* **Proposer:** mmagician
+* **Your Project(s):** https://mmagician.github.io/
+* **Projects you think this work could be useful for** Any parachain team
+
+## Project Description :page_facing_up: 
+
+This is meant as a companion repository to [polkadot-validator-setup](https://github.com/w3f/polkadot-validator-setup), which allows anyone to launch a validator node in an automated and secure fashion.
+
+I would like to standardize (as much as possible) the process of spinning up new collator nodes for different parachains.
```

Comment:
See also https://github.com/w3f/Grants-Program/pull/693, Phala are working on a similar project based on [`polkadot-deployer`](https://github.com/w3f/Grants-Program/pull/693).

Reply:
Fyi according to Alessio `polkadot-deployer` and `polkadot-validar-setup` do serve a very similar purpose, the main difference between them indeed being that the first is Kubernetes-based while the other is meant to be installed on an Ubuntu OS.